main/acle.md

```diff
@@ -2937,6 +2948,18 @@ inclusive. See implementation documentation for the effect (if any) of
 this instruction and the meaning of the argument. This is available only
 when compiling for AArch32.
 
+``` c
+  uint64_t __chkfeat(uint64_t);
```

Comment:
The behaviour of this intrinsic is opposite that of the instruction (chkfeat instruction clears the bit if the feature is enabled), which I think is unique among intrinsics. It would be worthwhile specifically noting this.

Also "feature is available" should be "feature is enabled" (as the chkfeat instruction will leave the bit alone if the feature is available but disabled).

Reply:
I'd prefer to write/read this code in applications:
`if( __chkfeat(_CHKFEAT_GCS)) {
   // do GCS stuff.
}`
chkfeat must clear the bit if the feature is present due to it is a NOP on old CPUs where it won't change the register content. 
That logic twist is not required for the app developers. 


+1 for the 

> Also "feature is available" should be "feature is enabled"